services/services.rst

```diff
@@ -0,0 +1,209 @@
+Overview of services
+******
+
+Below are tools and resources that are used in the PFS project. 
+Many services are hosted at IPMU under `pfs.ipmu.jp <https://pfs.ipmu.jp>`_ 
+(mainly for instrument hardware development) or 
+`pfspipe.ipmu.jp <pfspipe.ipmu.jp>`_ (for software development), and also PFS 
+project is using several external services, like 
+`GitHub Subaru-PFS organization <https://github.com/Subaru-PFS>`_. 
+You may need account(s) and access to some or all of them depending on your 
+roles and contributions to the project, most of services hosted at IPMU are 
+under single sign on (SSO) by pfs.ipmu.jp. 
+Refer `account policy <account.rst>`_ for details and registration. 
+
+* `SuMIRe/PFS PBworks wiki`_
+* `Public Web Site at pfs.ipmu.jp`_
+* `PFS project internal website (pfs.ipmu.jp)`_
+* `Online meeting`_
+* `Services for software development at pfspipe.ipmu.jp and external`_
+
+SuMIRe/PFS PBworks wiki
+------
+
+"SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
+
+======
+Login and ToC
+======
+
+After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
+you will get a project index page named 
+"Subaru Measurement of Images and Redshifts (SuMIRe)". 
+This page contains links to contents in the wiki, such as 
+
+* PFS Project Office (General links, including link to telecon indexes)
+* Documents (Link list to documents)
+* Meeting, conference, etc.  (Link list to meetings)
+* Mailing lists (List of avail lists)
+* PFS working groups (Member list)
+
```

Comment:
Looking the latest top page, I'd like to suggest to reorganize this items. Here is just an example:

* Upcoming meetings and events
* Regular telecons
* Technical discussion items
* Science discussion items
* Member list of the PFS working group
* Other useful pages (index of face to face meeting, project web/blog site etc)

Reply:
let me left for 3rd round